src/components/storage/ReclaimSpaceModal.jsx

```diff
@@ -238,7 +238,7 @@ const getDeviceRow = (disk, devices, level = 0, unappliedActions, setUnappliedAc
     const device = devices[disk];
     const isDisk = device["is-disk"].v;
     const isPartition = device.type.v === "partition";
-    const typeLabel = isPartition ? (device.misc.fsLabel || device.misc.partName) : "";
+    const typeLabel = device.partition-label.v;
```

Comment:
I think it's partition-type-name

Reply:
It will be something in attrs dictionary, but i cant find it there.